File: program_data/github_java_sort_function/9/34.java
private static void selectionsort(double tablica[], int ile_liczb) {
    int min,i,j;
    double temp;
    for (i=0;i<ile_liczb-1;i++)  {
    min=i;
    for (j=i+1;j<ile_liczb;j++)
    if (tablica[j]<tablica[min])
    min=j;
    temp=tablica[min];
    tablica[min]=tablica[i];
    tablica[i]=temp;
    }
}

File: program_data/PKU_raw/104/34.c
int main()
{
	int i,a,b,temp,t1,t2;
	scanf("%d %d",&a,&b);
	if(a<b){temp=a;a=b;b=temp;}
	t1=(int)(log(a)/log(2));t2=(int)(log(b)/log(2));
	for(i=t1;i>t2;i--){a=a/2;}
	for(;a!=b;)
	{a=a/2;b=b/2;}
	printf("%d\n",a);
	return 0;
}